src/oop/user_defined_classes/medium/q5/MainTest.java

```diff
@@ -0,0 +1,69 @@
+package oop.user_defined_classes.medium.q5;
+
+import global.ObjectTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.tools.CustomAssertions._assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MainTest {
+    // Java
+    private final String className = "Bank";
+    private final String var = "fee";
+    private final String methodName = "increase";
+    private final double initialFee = 5.25;
+
+    private ObjectTest classInstance;
+
+    @BeforeEach
+    public void setUp() throws Throwable {
+        String classString = "oop.user_defined_classes.medium.q5." + className;
+        classInstance = new ObjectTest(classString);
+        String modifiedClassMessage =
+                "You have modified the provided portions of class " + className + ". Please revert them to the original state.";
+        assertTrue(classInstance.hasField(var, double.class), modifiedClassMessage);
+        assertTrue(classInstance.hasModifier(var, "private"), modifiedClassMessage);
+
+    }
+
+    @Test
+    void hasCorrectConstructor() throws Throwable {
+        String incorrectCons = "Your " + className + " class does not have the correct constructor.";
```

Comment:
Can you change the wording on these so the student has a better idea of what you mean? I think this is for checking the constructor parameters, so say that, please.